src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public String getRandomBall () {
+        Ball ball = new Ball();
+        ball.color = new ColorSupplier().getRandomColor().toString();
```

Comment:
[to get string representation of enum use `name()` instead of `toString()`](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants)